app/views/taxon_changes/_show_change_taxon.html.haml

```diff
@@ -15,23 +15,31 @@
           = link_to t(:update_your_content), taxon_change_commit_for_user_path(taxon_change), :class => "btn btn-default"
       - if taxon_change.editable_by?(current_user)
         = link_to t(:edit), edit_taxon_change_path(taxon_change), :class => "btn btn-default"
-        - if show_commit_button && !taxon_change.committed? && taxon_change.committable_by?( current_user )
```

Comment:
This is what I was referencing above. We should ensure there is no way to make a TaxonChange with a non-null value of committed_on have a status of draft (or anything other than committed)